dvc/logger.py

```diff
@@ -145,12 +145,9 @@ def emit(self, record):
             if record.exc_info:
                 _, exc, *_ = record.exc_info
                 if hasattr(exc, "__pretty_exc__"):
-                    try:
-                        self.emit_pretty_exception(exc, verbose=_is_verbose())
-                        if not _is_verbose():
-                            return
-                    except Exception:  # noqa: BLE001, S110
-                        pass
+                    self.emit_pretty_exception(exc, verbose=_is_verbose())
+                    if not _is_verbose():
+                        return
```

Comment:
I don't trust the handling in `strictyaml` enough, that this should still fallback to other messages.

Reply:
> I don't trust the handling in `strictyaml` enough, that this should still fallback to other messages.

Thanks for raising your concerns about the code. I understand your point that we should consider including a fallback option for situations where the strictyaml module might not work correctly. To address this issue, I've created a separate ticket (#10201) so that we can discuss and implement a solution without muddying the current pull request with unrelated changes. Let's continue the conversation there.